webrtc.html

```diff
@@ -3885,19 +3901,48 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
                   </ul>
                 </li>
                 <li>
-                  <p>If <var>sender</var> was left unset by the previous step,
-                  create a new <code><a>RTCRtpTransceiver</a></code> with
-                  <code>sender.track</code> set to <var>track</var>, add it to
-                  <var>connection</var>'s set of transceivers.</p>
-                  <p>Let <var>sender</var> be the
-                  <code><a>RTCRtpSender</a></code> object associated with the
-                  newly created <code><a>RTCRtpTransceiver</a></code>.</p>
-                  <p>The initial value of an
-                  <code><a>RTCRtpTransceiver</a></code>'s <code><a data-for=
-                  "RTCRtpTransceiver">mid</a></code> attribute is null. Setting
-                  a new <code><a>RTCSessionDescription</a></code> may change it
-                  to a non-null value, as defined in <span data-jsep=
-                  "processing-a-local-desc processing-a-remote-desc">[[!JSEP]]</span>.</p>
+                  <p>If <var>sender</var> was set by the previous step, run the
+                  following steps to use that sender:</p>
+                  <ol>
+                    <li>
+                      <p>Set <var>sender.track</var> to <var>track</var>.</p>
+                    </li>
+                    <li>
+                      <p>Set <var>sender</var>'s
+                      [[<a>associated MediaStreams</a>]] to
+                      <var>streams</var>.</p>
+                    </li>
+                    <li>
+                      <p>Enable sending direction on the <code>
```

Comment:
The intention is to go from recvonly to sendrecv and inactive to sendonly. Perhaps that needs to be defined (by JSEP)?